themes/vue/source/css/_team.styl

```diff
@@ -2,135 +2,135 @@
 
 #team-members
   .sort-by-distance-button
-    display: inline-block
-    padding: .4em .7em .45em
-    font-weight: bold
-    font-size: .5em
-    text-transform: uppercase
-    line-height: 1
-    border: none
-    background: $medium
-    color: white
-    border-radius: 3px
-    position: relative
-    cursor: pointer
-    float: right
-    margin-top: .3em
+    display inline-block
+    padding .4em .7em .45em
+    font-weight bold
+    font-size .5em
+    text-transform uppercase
+    line-height 1
+    border none
+    background $medium
+    color white
+    border-radius 3px
+    position relative
+    cursor pointer
+    float right
+    margin-top .3em
     i
-      margin-right: .25em
+      margin-right .25em
       &:last-child
-        margin-right: 0
+        margin-right 0
     &[disabled]
-      opacity: .7
-      cursor: default
+      opacity .7
+      cursor default
   .vuer
-    display: flex
-    padding: 25px 0
-    border-bottom: 1px dotted #ddd
+    display flex
+    padding 25px 0
+    border-bottom 1px dotted #ddd
     &:first-of-type
-      margin-top: 15px
+      margin-top 15px
     &:last-of-type
-      border-bottom: none
+      border-bottom none
     .avatar
-      flex: 0 0 80px
+      flex 0 0 80px
       img
-        border-radius: 50%
+        border-radius 50%
     .profile
-      padding-left: 26px
-      flex: 1
+      padding-left 26px
+      flex 1
       h3
-        margin: 0
-        font-size: 1.3em
+        margin 0
+        font-size 1.3em
         &::before, &::after
-          display: none
+          display none
         & > sup
-          text-transform: uppercase
-          font-size: .7em
-          letter-spacing: .3px
-          padding: 2px 5px
-          margin-left: 10px
-          color: transparentify($dark, #f9f7f5, .6)
-          background: #f9f7f5
-          border-radius: 5px
+          text-transform uppercase
+          font-size .7em
+          letter-spacing .3px
+          padding 2px 5px
+          margin-left 10px
+          color transparentify($dark, #f9f7f5, .6)
+          background #f9f7f5
+          border-radius 5px
       // NOTE: Removing the little signal icon for now
       // .distance
-      //   position: relative
+      //   position relative
```

Comment:
Yes, they are also stylus codes. Maybe some day they would be enabled again. :-)
(Is that OK?)